Turbolinks/WebView.swift

```diff
@@ -40,6 +40,11 @@ class WebView: WKWebView {
         if #available(iOS 11, *) {
             scrollView.contentInsetAdjustmentBehavior = .never
         }
```

Comment:
Ideally, I think we can remove `if #available(iOS 11, *)` now that we only support iOS 14+